src/components/intro-step-counter/intro-step-counter-component.vue

```diff
@@ -0,0 +1,26 @@
+<style lang="scss" scoped>
+@import './intro-step-counter-component.scss';
+</style>
+
+<template>
+  <div class="intro-steps">
+    <span :class="{'dot': true, 'filled': step === 1}" />
```

Comment:
We may be able to make this a little more dynamic if we passed in the `numberOfSteps` along with the current step number.